executor/executor.cc

```diff
@@ -422,6 +426,8 @@ void parse_env_flags(uint64 flags)
 		flag_sandbox = sandbox_setuid;
 	else if (flags & (1 << 3))
 		flag_sandbox = sandbox_namespace;
+	else if (flags & (1 << 4))
+		flag_sandbox = sandbox_android_untrusted_app;
 	flag_enable_tun = flags & (1 << 4);
```

Comment:
It seems then that this is a really fragile way to pass the flags.  Can we not use bitfields here, which handle ordering automagically?

Reply:
The fragility comes from the fact that this is inter-language communication, so either have to have inherent duplication or a code generation. 
Bitfields should make it better. So if you feel like it, go ahead and change it to bitfields.

Reply:
Yeah, I was really surprised by the choice to do manual IPC instead of Protobufs / CapnProto / MessagePack / or even JSON.  It seems that the IPC code shouldn't be on the fast path and could afford any of these.

Reply:
Good luck finding any native library that won't cause brain damage while integrating into build system, is maintained and is portable anywhere from windows to akaros. On top of that we need it to work on gcc4.5+, mingw and clang. Don't use malloc, don't use exceptions, don't use libstdc++.
If you have 30 full time engineers on the side supporting all that for you, it may be an easy decision to "just use a native library". But not so in the wild.
Instead of all that we just stamp uint64 into memory. You will need to do slightly more justification to convince me to import a native library to pass 8 bytes ;)

Reply:
Done